### PR TITLE
#167728004 Complete sequelize setup using sequelize-cli

### PR DESCRIPTION
**Pull request by richienabuk**
**What does this PR do?**

Complete sequelize setup using sequelize-cli

**How should this be manually tested?**

- Clone repo and run `npm install`
- Add dotenv package and add the database variable to the .env file, see config/database.json
- Run `/node_modules/.bin/sequelize db:migrate` to migrate dummy user migration

**Any background context you want to provide?**

- Sequelize ORM is required in the project specification to be used with Postgresql.

**What are the relevant stories?**

[#167728004](https://www.pivotaltracker.com/n/projects/2381984/stories/167728004)

**Screenshots (if appropriate)**

- None

### CODE REVIEW

**Review by Abobos — APPROVED** [review]

**Review by dieudonneAwa — APPROVED** [review]

File: src/config/database.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "development": {
```
Comment (chukwuemekachm) [inline src/config/database.json:2]: Can we use the envs as the suffix for the `DATABASE_URL` in lieu of prefixes here?
Eg
```json
DATABASE_URL_DEV
DATABASE_URL_TEST
```
Update the `.env.example` file with any new env vars.

**Review by chukwuemekachm — CHANGES_REQUESTED** [review]
Nice work @richienabuk.
Just one more thing before we merge.

File: src/config/database.json (hunk 1 above)
Comment (richienabuk) [reply src/config/database.json:2]: I've done that.
Thanks
@chukwuemekachm

**Review by chukwuemekachm — APPROVED** [review]
LGTM.

**Pull request merged**